install/system.php

```diff
@@ -131,7 +131,7 @@ private function config()
 		$data['keywords'] = $_POST['keywords'];
 		$data['description'] = $_POST['description'];
 		$data['analytics'] = ($_POST['analytics']) ? $_POST['analytics'] : false;
-		$data['cdn'] = ($_POST['cdn'] == "yes") ? true : false;
+		$data['cdn'] = ($_POST['cdn'] == '1') ? true : false;
```

Comment:
I believe we have to observe this as it goes into effect. I dont feel right about this specific part.

Both "1" and "true : false" feels out of place and or uneccessary to me. (i could be wrong tho)

Reply:
Other than that i see no reason to not merge this and put it to the test.

Well done and thank you for your contribution

//Merged.